README.md

```diff
@@ -36,7 +36,7 @@ You don't have to worry about configuration or scaling up computational resource
 1. Prepare local infrastructure
 ```shell
 docker-compose pull
-docker-compose up
+VERSION=nightly docker-compose --profile jupyter --profile observability up
```

Comment:
I think we could add a "full" profile (something like `profiles: ["jupyter", "full"]` in the compose file, which would drop a profile flag in the call: `VERSION=nightly docker-compose --profile full up`

(maybe "all" a the better choice than "full"? don't have a strong preference either way)

Reply:
I like `full` more :)